app/src/main/java/com/talkable/demo/MainActivity.java

```diff
@@ -147,9 +147,11 @@ public void onError(ApiError error) {
     }
 
     private Customer getCustomer() {
-        Customer customer = new Customer();
+        Customer customer = null;
+        HashMap<String, String> customProperties = new HashMap<String, String>();
+        customProperties.put("prop1", "value2");
         try {
-            customer.setEmail(getEmail());
+            customer = new Customer(null, null, null, getEmail(), customProperties);
```

Comment:
You can use constructor without attributes or with just email and then use setters. I think it is more clear than passing `null`.